tests/integration/test_lb_interface.py

```diff
@@ -10,15 +10,16 @@
 async def test_build_and_deploy(ops_test, lb_charms):
     lb_lib_path = await ops_test.build_lib(".")
     lb_charms._lb_lib_url = lb_lib_path
-    bundle = ops_test.render_bundle(
-        "tests/integration/bundle.yaml",
-        charms=await ops_test.build_charms(
-            lb_charms.lb_provider,
+    charms: dict = await ops_test.build_charms(lb_charms.lb_provider)
+    charms.update(
+        **await ops_test.build_charms(
```

Comment:
this adjustment was made when it was observed integration tests failed due to charmcraft not permitting two charms to build simultaneously